.github/workflows/python-package-release.yml

```diff
@@ -0,0 +1,34 @@
+name: Python package
+
+on: 
+    push:
+        tags:
+            - v*
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
I could imagine that you could find containers here that have pip and python already installed, which could speed things up.

Reply:
setup-python seems to be what pretty much everyone does: https://github.com/actions/setup-python. I think it's doing some caching anyway, seems pretty quick to start.